eng/testing/performance/ios_scenarios.proj

```diff
@@ -30,5 +30,17 @@
         <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
         <PostCommands>$(Python) post.py</PostCommands>
     </HelixWorkItem>
+    <HelixWorkItem Include="SOD - Maui iOS .app Size" Condition="'$(iOSLlvmBuild)' == 'False'">
+      <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
+      <PreCommands>cd $(ScenarioDirectory)mauiios;xcopy %HELIX_CORRELATION_PAYLOAD%\MauiiOSDefault .\app\/e;$(Python) pre.py</PreCommands>
```

Comment:
Is there a reason that we don't do the copying inside of the pre.py command? Seems like that would follow the pattern that we have been using of doing precommand work in pre.py when possible

Reply:
For now, I followed what was already setup for the Mono SOD.